File: Algoritmos Computacionales/esfera.py
import numpy as np 
print("Calcularemos el volumen de una esfera\nCuál es el radio r de la esfera?")
r=float(input())
print("la clase de r es:",type(r))
v=(4/3)*np.pi*(r**3)
print("El volumen es: ",end="")
print(v)

File: Algoritmos Computacionales/esfera.jl
println("Calcularemos el volumen de una esfera")
println("Cuál es el radio r de la esfera?")
r=parse(Float64,readline())
println("la clase de r es: ",typeof(r))
v=(4/3)*pi*(r^3)
print("El volumen de la esfera es: ")
println(v)